lib/ansible/modules/packaging/language/pip.py

```diff
@@ -125,6 +127,21 @@
     name: bottle
     version: 0.11
 
+# Install (bottle) python package with version specifiers (only in 2.7)
+- pip:
+    name: bottle
+    version: '>0.10,<0.20,!=0.11'
+
+# Install multi python packages with version specifiers (only in 2.7)
+- pip:
+    name:
+      - 'django>1.11.0,<1.12.0'
+      - 'bottle>0.10,<0.20,!=0.11'
+
+# 2.7 also support multi packages in one name string
```

Comment:
Please remove this example, since I would consider this usage as not recommended.